Templates/MonoGame.Templates.CSharp/content/MonoGame.2D.StartKit/___SafeGameName___.Core/Content/Fonts/Hud.spritefont

```diff
@@ -11,7 +11,7 @@ with.
     <!--
     Modify this string to change the font that will be imported.
     -->
-    <FontName>Tahoma</FontName>
+    <FontName>Arial</FontName>
```

Comment:
I remember this statement used to be in the XNA archive:
>Note that fonts are licensed and not sold, and that most commercial fonts, and most fonts supplied with Microsoft products such as Windows and Office, are not licensed for redistribution with games or other applications. It is your responsibility to make sure any fonts you include with your games are properly licensed for redistribution.

The main issue is that free fonts may allow use within documents/pictures/etc, but there are usually embedding & modifying restrictions, and `SpriteFont` XNB files are essentially embedded fonts in a modified bitmap format.

There was the "Redistributable Font Pack for XNA Game Studio" (https://github.com/SimonDarksideJ/XNAGameStudio/tree/archive/Samples/RedistributableTTFs_ARCHIVE_3_1/RedistributableTTFs), but even that license isn't clear if these are usable or for "XNA Game Studio" use only.

It seems Google Fonts (https://fonts.google.com) have "Open Font License" fonts that allow embedding, not used them myself though.

Reply:
@squarebananas we are discussing how best to support unicode characters internally, especially now that Microsoft no longer distribute the Arial Unicode MS TTF.

Though the licences aren't 100% clear I've yet to hear of any game Devs, even as far back as XNA 3.x, being sued for using an xnb version of their font. Always worth remembering than an xnb is usually a small subset of the full TTF. So it's not usually a 1 for 1 likeness and that's why I think game Devs aren't being chased. But it's definitely worth keeping an eye on the situation.

Reply:
Yeah, I haven't heard of any issues either. I raised it more because this sample will be within the MonoGame repository, and the MonoGame licenses imply any users will have fairly open distribution/publishing rights of the code/assets (https://github.com/MonoGame/MonoGame/blob/develop/LICENSE.txt).

Reply:
We are not embedding the .ttf in the xnb, we are rendering to a texture. That is allowed otherwise people who use fonts for artwork or images would be paying licenses per image download not just a one off for using the .ttf in the first place. 

This is someone people forget about this when they want MG to support direct font rendering.

Reply:
Thanks for the response. Using fonts within documents & images should be permitted, but I don't think commercial companies would let that loophole exist for other usages. Especially as various bitmap formats were used before vector based ones were introduced.

https://learn.microsoft.com/en-us/typography/fonts/font-faq
>If I convert the font into a bitmap font can I include that in my game or app?
>>No, converting Windows fonts to other formats does not change the rules around embedding or redistribution, and format conversion itself is not allowed. Many Microsoft supplied fonts are available for app and game licensing through the original font foundry or Monotype.

https://learn.microsoft.com/en-us/previous-versions/windows/xna/bb464059(v=xnagamestudio.42)
>Font licenses vary from vendor to vendor, but most don't allow redistribution of the fonts, and that includes redistribution of reproductions such as bitmaps containing the rasterized character set.

Reply:
Yup sorry getting mixed messages. If you want to use a font in your game then yes you need a license to use the font in your game.  
I think what I was getting at was you probably don't need to pay a per install license ever time your game is installed.

Reply:
You could use Noto Sans/Open Sans that have a permissive license. AFAIK

Reply:
@viniciusjarina the team is investigating a font fall back system, particularly when it comes to multi-byte character sets.

Reply:
I see. NotoSansCJK covers pretty much all the majos languages characters.  Latin, Cyrilic, Chinese, Japanese, Korean https://github.com/notofonts/noto-cjk that's what I have been using

Reply:
@viniciusjarina that maybe what we need to fall back to, as `Arial Unicode MS` is no longer being distributed or updating by Microsoft.